api/internal/filter/authentication.go

```diff
@@ -41,8 +42,8 @@ func Authentication() gin.HandlerFunc {
 				"message": "Request Unauthorized",
 			}
 
-			if err != nil || !token.Valid {
-				log.Warnf("token validate failed: %s, %v", err, token.Valid)
+			if err != nil || token == nil || !token.Valid {
+				log.Warnf("token validate failed: %s, %v", err, false)
```

Comment:
Hi, I think `false` is not necessary to log.

Reply:
@starsz  Thanks. It's to keep log format that same as before. How about change this to

```suggestion
				log.Warnf("token validate failed: %s", err)
```

Reply:
@starsz Done and PR renewed, thanks for your suggestion.